### PR TITLE
Lesson 12

### PR DESCRIPTION
**Pull request by vankampenc**

### CODE REVIEW

**Comment by vankampenc** [comment]
Added to the incorrect repo. Closing this PR

File: CSS/index.css [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+    i {
+        font-size: 1.65rem;
+    }
+}
+
+
+/* ID Section Scroll Adjuster */
+
+#About, #Experience, #Connect, #Skills, #Projects {
+    scroll-margin-top: 120px;
```
Comment (mannsn) [inline CSS/index.css:196]: Would avoid px as a unit and instead use something like 5 em

File: CSS/index.css [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+        padding: 3px 9px;
+        font-size: .75rem;
+    }
+}
+
+/* Footer */
+
+footer {
+    background-color: black;
+    padding: 300px 30px;
```
Comment (mannsn) [inline CSS/index.css:163]: This makes a really big black box at the bottem.  Again would avoid using px but use relative sizing.

File: CSS/index.css [hunk 3]
```diff
@@ -0,0 +1,197 @@
...
+    margin-top: 20px;
+    margin-left: auto;
+    margin-right: auto;
+    width: 500px;
+    border: 20px solid #3f5db7;
+}
+
+/* Tablet */
+
+@media (max-width:1024px) {
```
Comment (mannsn) [inline CSS/index.css:45]: Was a little hard to see what was changing, so I actually tried your query with different border colors - and it does work properly.

File: CSS/index.css [hunk 4]
```diff
@@ -0,0 +1,197 @@
...
+
+.sticky {
+    position: fixed;
+    top: 0;
+    width: 100%
+}
+
+/* Tablet Header */
+
+@media (max-width:1024px) {
```
Comment (mannsn) [inline CSS/index.css:131]: Again, a bit hard to see what your queries were visually doing to the screen, but they do work.

File: CSS/index.css [hunk 5]
```diff
@@ -0,0 +1,197 @@
...
+.boldFont {
+    font-weight: bolder;
+}
+
+/* Header */
+
+nav {
+    display: flex;
+    gap: 10px;
+    justify-content: flex-end;
```
Comment (mannsn) [inline CSS/index.css:93]: For larger screens going to the end doesn't match with the rest of the layout.

**Review by mannsn — COMMENTED** [review]
Overall, very nice. Marking assignment as successful.   I would avoid using px units and instead use .rem units or %.   It did seem to work though.   The flex box is working well unless you to a very small size (then there is overflow).  Would put connect below skills and projects.   Would remove big black footer.

File: CSS/index.css (hunk 1 above)
Comment (vankampenc) [reply CSS/index.css:196]: Updated

File: CSS/index.css (hunk 3 above)
Comment (vankampenc) [reply CSS/index.css:45]: My goal was to make it look similar 😅. Sorry if it was difficult to see the changes!

File: CSS/index.css (hunk 4 above)
Comment (vankampenc) [reply CSS/index.css:131]: Resolved

File: CSS/index.css (hunk 2 above)
Comment (vankampenc) [reply CSS/index.css:163]: I created a large footer so the page nav scroll would jump correctly to the last section.

File: CSS/index.css (hunk 5 above)
Comment (vankampenc) [reply CSS/index.css:93]: I thought it made sense to have my nav bar always set to the upper right, but am open to other ideas that make sense :)

**Pull request closed without merging**